src/main/java/nl/jessicavdz/redlightgreenlight/Block/BlockBreakCancel.java

```diff
@@ -0,0 +1,17 @@
+package nl.jessicavdz.redlightgreenlight.Block;
+
+import org.bukkit.entity.Player;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+import org.bukkit.event.block.BlockBreakEvent;
+
+public class BlockBreakCancel implements Listener {
+
+    @EventHandler
+
+    public void onBlockBreak(BlockBreakEvent event) {
+        Player player = event.getPlayer();
+        if(player.isOp()) return;
```

Comment:
It is better not use return here, but rather check if the player is not OP (!player.isOp()) and than execute the setcancelled if that's the case